### PR TITLE
modified some bad code

### PR DESCRIPTION
**Pull request by vishalvide83**

### CODE REVIEW

File: Project/p1.py [hunk 1]
```diff
@@ -1 +1,2 @@
 edit by owner
+new edit by contributerr/forker
```
Comment (vishalvide) [inline Project/p1.py:2]: add some more efficient code

File: Project/p1.py [hunk 2]
```diff
@@ -1 +1,2 @@
 edit by owner
```
Comment (vishalvide) [inline Project/p1.py]: Here change the spelling of "owner"

Comment (vishalvide83) [reply Project/p1.py]: Ok

File: Project/p1.py (hunk 1 above)
Comment (vishalvide83) [reply Project/p1.py:2]: OK

**Comment by vishalvide83** [comment]
see now I have successfully changed the spelling of owner

**Pull request merged**